kratos/processes/calculate_discontinuous_distance_to_skin_process.cpp

```diff
@@ -40,18 +40,9 @@ namespace Kratos
 		// Initialize the intersected objects process
 		mFindIntersectedObjectsProcess.Initialize();
 
-		// Reset the nodal distance values
-		const double initial_distance = 1.0;
-
-		#pragma omp parallel for
-		for (int k = 0; k< static_cast<int> (mrVolumePart.NumberOfNodes()); ++k) {
-			ModelPart::NodesContainerType::iterator itNode = mrVolumePart.NodesBegin() + k;
-			itNode->Set(TO_SPLIT, false);
-			itNode->GetSolutionStepValue(DISTANCE) = initial_distance;
-		}
-
 		// Reset the Elemental distance to 1.0 which is the maximum distance in our normalized space.
 		// Also initialize the embedded velocity of the fluid element and the TO_SPLIT flag.
+		const double initial_distance = 1.0;
```

Comment:
for my understanding: this is then scaled to the max distance in the model?

Reply:
No. For a non-intersected element the `ELEMENTAL_DISTANCES` will be `(1,1,1)` (or `(1,1,1,1)` for a tetra). In any case, this is the discontinuous distance algorithm, for which this initial shouldn't make any difference in the final level set.

Reply:
i think this should be at least larger than the largest element size

Reply:
We can set this value if you prefer it but it shouldn't make any difference. The elemental distance is computed element by element only if there are intersections. The point is that the continuous one is done somehow globally by looping the nodes. 

In any case this PR deals with de continuous level set behaviour so I wouldn't mix it. I only modified this file since the changes in here were a leftover from the continuous distance migration.

Reply:
point is that 1 may be large or small depending on the element size, so i would have used something which we can measure. your call anyway

Reply:
I'd rather have something that is the same for all the non-intersected elements since this number makes no difference and simplifies eventual debugging of corner cases (you only focus on the values different to the default one). If you don't like 1.0 we can set the same as we are setting for the continuous distance case (the characteristic domain length).

Reply:
i woul definitely prefer the characteristic l nght than the 1.0...

Reply:
OK. So I'd implement the characteristic length calculation in the base discontinuous distance class to call it from both processes.